tree/dataframe/test/dataframe_helpers.cxx

```diff
@@ -682,3 +682,57 @@ TEST(RunGraphs, AlreadyRun)
    ROOT_EXPECT_WARNING(ROOT::RDF::RunGraphs({r1, r2, r3, r4}), "RunGraphs",
                        "Got 4 handles from which 2 link to results which are already ready.");
 }
+
+TEST(RDFHelpers, ProgressHelper_Existence_ST)
+{
+   // Redirect cout.
+   std::streambuf *oldCoutStreamBuf = std::cout.rdbuf();
+   std::ostringstream strCout;
+   std::cout.rdbuf(strCout.rdbuf());
+   auto d_write_1 = ROOT::RDataFrame(10000000).Define("x", "42").Snapshot("tree", "f1.root");
+   auto d_write_2 = ROOT::RDataFrame(10000000).Define("y", "1").Snapshot("tree", "f2.root");
+   ROOT::RDF::RNode d = ROOT::RDataFrame("tree", {"f1.root", "f2.root"});
+   ROOT::RDF::Experimental::AddProgressbar(d);
+   d.Count().GetValue();
+   // Restore old cout.
+   std::cout.rdbuf(oldCoutStreamBuf);
+
+   EXPECT_FALSE(strCout.empty());
```

Comment:
The wonders of standard C++ library :smiling_face_with_tear:  For `std::ostringstream` we first need a conversion to `std::string`  
```suggestion
   EXPECT_FALSE(strCout.str().empty());
```